api-participant/src/main/java/bio/terra/pearl/api/participant/service/AuthUtilService.java

```diff
@@ -16,32 +17,45 @@
 @Service
 public class AuthUtilService {
   private EnrolleeService enrolleeService;
+  private EnrolleeRelationService enrolleeRelationService;
   private PortalService portalService;
   private PortalParticipantUserService portalParticipantUserService;
 
   public AuthUtilService(
       EnrolleeService enrolleeService,
       PortalService portalService,
-      PortalParticipantUserService portalParticipantUserService) {
+      PortalParticipantUserService portalParticipantUserService,
+      EnrolleeRelationService enrolleeRelationService) {
     this.enrolleeService = enrolleeService;
     this.portalService = portalService;
     this.portalParticipantUserService = portalParticipantUserService;
+    this.enrolleeRelationService = enrolleeRelationService;
   }
 
   /**
    * returns the enrollee if the user is authorized to access/modify it, throws an error otherwise
    */
   public Enrollee authParticipantUserToEnrollee(UUID participantUserId, String enrolleeShortcode) {
-    // for now, a user is only allowed to access an enrollee if it's themself.  Later, we'll add
-    // proxies
     Optional<Enrollee> enrolleeOpt =
         enrolleeService.findByEnrolleeId(participantUserId, enrolleeShortcode);
     if (enrolleeOpt.isEmpty()) {
-      throw new PermissionDeniedException("Access denied for %s".formatted(enrolleeShortcode));
+      return authParticipantUserToGovernedEnrollees(participantUserId, enrolleeShortcode);
     }
     return enrolleeOpt.get();
   }
 
+  private Enrollee authParticipantUserToGovernedEnrollees(
+      UUID participantUserId, String enrolleeShortcode) {
+    Optional<Enrollee> governedEnrolee =
+        enrolleeRelationService.findGovernedEnrollees(participantUserId).stream()
+            .filter(enrollee -> enrollee.getShortcode().equals(enrolleeShortcode))
+            .findAny();
```

Comment:
Instead of finding an optional, and then checking it, you can just use `orElseThrow`
```suggestion
            .findFirst().orElseThrow(() -> new PermissionDenied...;
```